src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -63,25 +76,51 @@ For how to find billing and usage information in the UI, see [Billing-related UI
 </Callout>
 
-## Data ingest calculation [#usage-calculation]
+## How data ingest is calculated [#usage-calculation]
 
-Want to understand how data ingest is calculated? See [Data ingest calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing).
+See [Data ingest calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing).
 
 Want to manage your data? See [Manage data](/docs/data-apis/manage-data/manage-data-coming-new-relic).
 
-## User count calculation [#user-count]
+## How billable users are calculated [#user-count]
 
-To learn how billable users are calculated, see [User count calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing).
+See [Billable user calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing).
 
 ## Billing schedule options [#usage-plans]
 
-A New Relic organization can have one of several types of plans that govern how billing scheduling is handled. Here are the main options:
+A New Relic organization can have one of several types of plans that govern how your billing is scheduled and implemented. Here are the main options:
 
 * **Pay-as-you-go**: With this option, billing happens at the end of each calendar month. There are no payment commitments past the current month and an organization can cancel at any time. An organization's first end-of-month bill includes billable usage starting from when their credit card information was input into New Relic.
 * **Commitment contracts**: Commitment contract options include our Volume Plan, Savings Plan, and other options. With commitment contracts, an estimate of long-term expected usage is created and there's an agreed upon payment commitment. If an organization's usage exceeds that commitment, the excess usage is billed at the previously agreed-upon rates. These are available primarily to Pro and Enterprise edition organizations. 
 
 For more about how these options can affect your user downgrade rules, see [Downgrade rules](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#user-downgrade-rules).
 
+## Understand order terms [#order]
+
+Your order and invoice contain several details about your pricing plan. Here's an example of an item found on a New Relic order: 
+
+```
+New Relic Volume Plan - Pro Full Platform Users
+```
+
+Here's a breakdown of these terms: 
+
+* The `New Relic` means you're on our [primary user billing version](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing/#pricing-versions). (If this is `New Relic One`, that means you're on our original user billing version.)
```

Comment:
@zuluecho9  the "The" at the start of these bullets seems odd and I think probably you can delete them and just start with the term itself.